contrib/librespot.service

```diff
@@ -9,7 +9,7 @@ User=nobody
 Group=audio
 Restart=always
 RestartSec=10
-ExecStart=/usr/bin/librespot -n "%p on %H"
+ExecStart=/bin/sh -c 'exec /usr/bin/librespot --name librespot@$(hostname -s)'
```

Comment:
Oh, the shell is actually needed because syntax like `$(hostname -s)` isn't accepted here, so it's slightly kind of a hack for the above bug.